src/main/java/org/cactoos/io/LoggingInputStream.java

```diff
@@ -43,6 +44,7 @@
  * @since 0.29
  */
 @SuppressWarnings({"PMD.LoggerIsNotStaticFinal", "PMD.MoreThanOneLogger"})
+@SuppressFBWarnings("RR_NOT_CHECKED")
```

Comment:
@ilyakharlamov the same above.

Reply:
The actual issue is fixed in 7cece161

Reply:
@ilyakharlamov why did we leave the suppression?

Reply:
@llorllale removed in a1411562